rlog.go

```diff
@@ -416,6 +571,14 @@ func basicLog(logLevel int, traceLevel int, format string, prefixAddition string
 	}
 }
 
+// Flush syncs the content of the logfile, which might still be in a memory
+// buffer, to storage. This should rarely be necessary during normal operation.
+func Flush() {
```

Comment:
To be honest, I don't even know if this is needed. I added it while debugging an issue, which ended up having a different cause.

I don't know if I can automate a Sync when a program panics or Exits. On Exit all IO buffers should be flushed anyway. I'm reading some vague hints that this happens on killed or paniced processes as well. I'll probably just remove this one, since it just seems confusing then.